st2common/st2common/cmd/install_pack.py

```diff
@@ -53,28 +56,37 @@ def _register_cli_opts():
             help="True to force pack installation and ignore install "
             "lock file if it exists.",
         ),
+        cfg.BoolOpt(
+            "get-dependencies",
+            default=False,
+            help="True to install pack dependencies",
+        ),
     ]
     do_register_cli_opts(cli_opts)
 
 
-def main(argv):
-    _register_cli_opts()
+def get_pack_dependencies(pack, verify_ssl, force, dependencies, proxy_config):
+    pack_path = get_pack_base_path(pack)
 
-    # Parse CLI args, set up logging
-    common_setup(
-        config=config,
-        setup_db=False,
-        register_mq_exchanges=False,
-        register_internal_trigger_types=False,
-    )
+    try:
+        pack_metadata = get_pack_metadata(pack_dir=pack_path)
+        result = pack_metadata.get("dependencies", None)
+    except Exception:
+        LOG.error("Could not open pack.yaml at location %s" % pack_path)
+        result = None
+    finally:
+        if result:
+            LOG.info('Getting pack dependencies for pack "%s"' % (pack))
+            download_packs(result, verify_ssl, force, dependencies, proxy_config)
+            LOG.info('Successfully got pack dependencies for pack "%s"' % (pack))
 
-    packs = cfg.CONF.pack
-    verify_ssl = cfg.CONF.verify_ssl
-    force = cfg.CONF.force
-
-    proxy_config = get_and_set_proxy_config()
 
+def download_packs(packs, verify_ssl, force, dependencies, proxy_config):
     for pack in packs:
+        if pack in listdir("/opt/stackstorm/packs"):
```

Comment:
is this path available as a config parameter?  I would hope we wouldn't have to hard code this.